recipes/glmnet_py/build.sh

```diff
@@ -0,0 +1,2 @@
+gfortran ./glmnet_python/GLMnet.f -fPIC -fdefault-real-8 -shared -o ./glmnet_python/GLMnet.so
```

Comment:
You also probably want to use something like ${FC:-gfortran} instead of gfortran directly